src/ui/windows/TogglDesktop/TogglDesktop/ui/views/EditView.xaml.cs

```diff
@@ -658,7 +661,7 @@ private bool tryCreatingNewProject(string text, string color)
             var ret = Toggl.AddProject(
                 this.timeEntry.GUID, this.selectedWorkspaceId,
                 this.selectedClientId, this.selectedClientGUID,
-                text, false, color) != null;
+                text, !isProjectPublicCheckBox.IsChecked.GetValueOrDefault(true), color) != null;

```

Comment:
```c#
.GetValueOrDefault(true)
```
I suggest that the default of `isProjectPublicCheckBox` should be a `false` because the project when creating is private project by default.

Reply:
I agree, fixed.
Although this is just a backup value, it will not be used as it is a 2-state checkbox that always returns `true` or `false`.